bin/build_project.sh

```diff
@@ -217,30 +217,33 @@ fi
 echo
 echo " -----Running make ---- " 
 
-# Workaround for race condition in the build of compiler-rt
-for prebuild in $(${AOMP_CMAKE} --build . --target help | sed -n '/-version-list/s/^... //p') ; do
-  ${AOMP_CMAKE} --build . -j $AOMP_JOB_THREADS --target $prebuild
-done
+if [ "$1" != "install-fast" ] ; then
+   # Workaround for race condition in the build of compiler-rt
+   for prebuild in $(${AOMP_CMAKE} --build . --target help | sed -n '/-version-list/s/^... //p') ; do
+       ${AOMP_CMAKE} --build . -j $AOMP_JOB_THREADS --target $prebuild
+   done
 
-# Required for amdllvm support
-echo ${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS clang lld compiler-rt
-${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS clang lld compiler-rt
+   # Required for amdllvm support
+   echo ${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS clang lld compiler-rt
+   ${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS clang lld compiler-rt
 
-# Required for amdllvm support
-echo ${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS runtimes cxx
-${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS runtimes cxx
+   # Required for amdllvm support
+   echo ${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS runtimes cxx
+   ${AOMP_CMAKE} --build . -- -j $AOMP_JOB_THREADS runtimes cxx
+fi
 
 # Finish building
+echo "Running CMAKE in ${PWD}"
 echo ${AOMP_CMAKE} --build . -j $AOMP_JOB_THREADS
 ${AOMP_CMAKE} --build . -j $AOMP_JOB_THREADS
 if [ $? != 0 ] ; then
    echo "ERROR make -j $AOMP_JOB_THREADS failed"
    exit 1
 fi
 
-if [ "$1" == "install" ] ; then
+if [ "$1" == "install" ] || [ "$1" == "install-fast" ] ; then
    echo " -----Installing to $INSTALL_PROJECT ---- "
-   $SUDO ${AOMP_CMAKE} --build . -j $AOMP_JOB_THREADS --target install
```

Comment:
can we keep the -j arg , in case folks want/need to limit jobs here ?

Reply:
cmake --install doesn't have a `-j` option. This doesn't do anything other than copy the built files to the install folder so I think it just runs with 1 thread.